frontend/src/constants/Routes.ts

```diff
@@ -7,3 +7,7 @@ export const SIGNUP_PAGE = "/signup";
 export const HOME_PAGE = "/home";
 
 export const VERIFICATION_PAGE = "/verify";
+
+export const SCHEDULING_PAGE = "/schedule";
+
+export const SCHEDULE_THANKYOU_PAGE = "/thankyou";
```

Comment:
should we route this as `/schedule/confirmation`
```suggestion
export const SCHEDULE_CONFIRMATION_PAGE = "/schedule/confirmation";
```
and then it doesn't have to be an `exact` path in Route